charts/kube-prometheus-stack/templates/prometheus/rules-1.14/k8s.rules.yaml

```diff
@@ -160,4 +160,14 @@ spec:
       labels:
         workload_type: statefulset
       record: namespace_workload_pod:kube_pod_owner:relabel
+    - expr: |-
```

Comment:
This file is generated, you'd need to submit this upstream in https://github.com/prometheus-operator/kube-prometheus